WireUI/Sources/WireReusableUIComponents/AccessoryTextField/AccessoryTextField.swift

```diff
@@ -154,12 +154,12 @@ extension AccessoryTextField {
 // MARK: - Custom edge insets
 
 extension AccessoryTextField {
-    public override func textRect(forBounds bounds: CGRect) -> CGRect {
+    override public func textRect(forBounds bounds: CGRect) -> CGRect {
```

Comment:
I suggest to configure the `modifierOrder` rule with a different order. At least `override` I would want to move to the right.